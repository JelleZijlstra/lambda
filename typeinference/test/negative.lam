let x = 0 - 42 in
print x
