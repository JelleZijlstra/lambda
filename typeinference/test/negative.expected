%%stdout%%
-42
Result: module
	let __result__ = ()
end
%%stderr%%
